### PR TITLE
Add/asciidoctor diagram

### LINKED ISSUE #27: UML Diagrams not working
See HackMD [feature description](https://hackmd.io/features#uml-diagrams). also amend `feature.adoc`

* Sequence Diagrams
* Flow charts
* Mermaid
* Graphviz
* Abc

### PR DESCRIPTION
**Pull request by ghost**
I fix #27 

I implement plantuml .But, It requires `:plantuml-server-url: http://www.plantuml.com/plantuml`.

But, It behaves perfectly.

image

### CODE REVIEW

**Comment by ghost** [comment]
Now, `:plantuml-server-url: http://www.plantuml.com/plantuml` not necessary.

**Comment by PReimers** [comment]
Sorry, I accidentally merged it already 😕

Did a miss click while reading the PR.

**Comment by ghost** [comment]
OMG.
By the way,Tell me if you need anything.

**Comment by madmas** [comment]
Okay, I reverted it for now. let's do a proper review.

File: public/js/index.js [hunk 1]
```diff
@@ -2803,7 +2803,18 @@ function updateViewInner () {
...
+
+  //extentions
+  //asciidoctor-plantuml
+  var plantuml = require('asciidoctor-plantuml');
+  plantuml.register(adoc.Extensions);
+
+  var adoc_options = Opal.hash2(['header_footer','attributes'],{ 'header_footer': true, 'attributes': ['icons=font@', 'showTitle=true']});
+
+  //add plantuml url
+  value = `:plantuml-server-url: http://www.plantuml.com/plantuml\n` + value;
```
Comment (madmas) [inline public/js/index.js:2815]: Do i undestand this correctly that it does add the plan-server-url implicitly to the header of each document? so the user does not need to add is manually on the one hand, but on the other hand, data is send to that server without notice?
I'd rather have a very explicit option. We could add the `:plantuml-server-url: http://www.plantuml.com/plantuml`in the first line when the user inserts [plantuml] or something similar, what do you think?

**Comment by madmas** [comment]
@asdfg0280  in this PR again: too many WIP commit messages. please take care for good, clean commit messages. Please have a look at https://chris.beams.io/posts/git-commit/ :)

Also, this PR changed back the Asciidoctor JS version - why?

Why is this addition of `~/.nvm` included in the PR?
bildschirmfoto 2018-07-27 um 09 16 58

@asdfg0280 please, cleanup the PR and recreate it.

File: public/js/index.js (hunk 1 above)
Comment (ghost) [reply public/js/index.js:2815]: Ahh,I think  `plantuml-server-url` is not necessary. cuz' plantuml.com is famous system.
So, I change implicit that.

**Comment by ghost** [comment]
sry, `.nvm` is my mistake.

**Pull request merged**